WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/ActivityLogListActivity.kt

```diff
@@ -53,10 +52,6 @@ class ActivityLogListActivity : LocaleAwareActivity(),
     }
 
     override fun onNegativeClicked(instanceTag: String) {
```

Comment:
Minor (🔍 ): It is generally suggested to avoid method with empty body. When that is indeed unavoidable, the best practice is adding an explicit comment like `// Do nothing.`. Wdyt?